product-input-fields-for-woocommerce.php

```diff
@@ -194,6 +194,9 @@ public function includes() {
 
 			if ( is_admin() ) {
 
+				require_once 'includes/class-pif-tracking-functions.php';
+				add_filter( 'ts_tracker_data', array( __CLASS__, 'pif_lite_ts_add_plugin_tracking_data' ), 10, 1 );
```

Comment:
@PratapDungrani Let's do the following:
1. Rename **class-pif-tracking-functions.php** to **class-alg-wc-pif-tracking.php**
2. Add the hook to the **class-alg-wc-pif-tracking.php** file.
3. Remove the class instantiation code from inside the class file itself and have it after the file has been loaded.